more rearranging and updating content

choldgraf opened the pull request:
This builds on the work that @GeorgianaElena was doing and does a bit more reorganization and updating of content. There are roughly two goals for this:

1. Make it easier for people to find relevant pages, and to streamline the grouping of pages so they're conceptually-similar.
2. Make it easier to _maintain_ pages by grouping things into folders that is close to how they're organized in the site.

In particular, the "Architecture" section has now been nested underneath a new "Jupyter Projects" page that is more like a "meta" page about the different projects.

This also introduces a few changes that will likely break some links. I'm assuming here that it's helpful to just "rip the band-aid" when it comes to re-organizing docs.

Would love to know what folks think!

choldgraf commented:
anybody have thoughts here? maybe @GeorgianaElena could give a quick review?

GeorgianaElena commented on docs/source/use/using.rst:
```diff
@@ -0,0 +1,172 @@
+=================
+Use Jupyter tools
```
What do you think about calling this section "About Jupyter tools" or just "Jupyter tools"?
"Use Jupyter tools" sounds a bit imperative in my head even without the exclamation mark.
I'm not a native speaker so that might be the cause :sweat_smile:

choldgraf replied:
hmmm - maybe "Using Jupyter tools"? I was trying to differentiate it as a section for "users" of Jupyter, as opposed to generally covering the jupyter community etc. But if you think it's clear-enough as "Jupyter Tools" I like that too. WDYT?

GeorgianaElena replied:
*Using Jupyter Tools* is perfect.

GeorgianaElena commented on docs/source/use/using.rst:
```diff
@@ -0,0 +1,172 @@
+
+   use-cases/content-user
+
+.. graphviz::
```
This part doesn't seem to render ok on CircleCI doc build :confused: 

![rendering-issue](https://user-images.githubusercontent.com/7579677/87328990-01153500-c53f-11ea-8196-00b618c94bb2.png)

choldgraf replied:
I think this should render fine in readthedocs, just not circleci...I had the same problem

GeorgianaElena replied:
Then maybe it's a good idea to enable the RTD PR build for this repo? :heart:

choldgraf replied:
I actually did and for some reason it isn't showing up, and I have no idea why haha...maybe the web hook is broken?

GeorgianaElena commented on docs/source/projects/architecture/content-architecture.rst:
```diff
@@ -135,3 +136,41 @@ IPython also includes a parallel computing framework,
+* :doc:`JupyterLab notebook model <lab:developer/notebook>`
+* :doc:`Design patterns in JupyterLab <lab:developer/patterns>`
+
+Projects overview
```
Maybe it would make more sense to have the visual overview at the very beginning of the section? What do you think?

choldgraf replied:
I agree in theory - though the reason I put it at the bottom was because it's a bit out-of-date and so doesn't quite match the content of the page itself. Not quite sure what to do about that but I didn't have the bandwidth to update this figure :-/

GeorgianaElena replied:
Then, I believe we should leave it like this until we update the figure. Definitely not block the PR because of it.

I'll happily work on updating it in the following days and move it at the begging of the section once we're happy with it. What do you think?

choldgraf replied:
sounds great - let me know if I can help you update the figure as well, happy to do a little virtual whiteboarding session or something

choldgraf replied:
I'll add a "TODO" so it's clear what should be done

choldgraf commented:
ah - re-syncing the webhook seems to have worked! I think I've addressed your comments @GeorgianaElena, wanna take a look at the RTD link and let me know what you think?

choldgraf commented:
Anything else you can think of @GeorgianaElena ? Maybe @parente has thoughts as well? I'm also fine w/ merging and iterating further

GeorgianaElena commented:
I gave another look at the RTD link and it looks really good :heart:
The only thing I spotted is that the left TOC doesn't show up on the community page. We could add the hidden toctree here too, like we did for the other pages that had this issue.

choldgraf commented:
ah good point! I've updated with a hidden toctree 👍 

anything else to take care of here?

GeorgianaElena commented:
LGTM :rocket:

choldgraf commented:
Sounds good! I'll plan to merge this in the next couple of days unless somebody suggests otherwise!

choldgraf commented:
ok - merging away, I'm happy to spot-check improvements if this turns up some broken links or something
